dotmailer/engagement-cloud-for-shopware
# [P2] Rebrand to dotdigital Engagement Cloud.

This PR changes all the visible instances of the old "dotmailer Email Marketin" to "dotdigital Engagement Cloud" and the logo in the menu item.
I've tried to keep the changes as unaggressive as possible, so there are things that could also change (like the css class names) but I thought this could be safer.

##How to test
 1 - First you need to have a local instance for shopware. You can download the code in their site and configure a wsl host to get it. 
 2 - Download the plugin from the plugin manager. This will create a folder in your file system ($storepath\engine\Shopware\Plugins\Community\Backend\DotmailerEmailMarketing\Views\backend\dotmailer_email_marketing)
3 - Remove the content of that folder and copy the code of this branch.
4 - Install the plugin in the plugin manager
5 - Activate it

The plugin name and the menu item should be now with the new brand.
